app/controllers/api/v1/admin/user/invoices_controller.rb

```diff
@@ -0,0 +1,31 @@
+class Api::V1::Admin::User::InvoicesController < Api::V1::BaseController
+  before_action :authenticate_user!, :load_user
+
+  def index
+    authorize :invoice, :index?
+    @invoice = @user.timelogs.start_time.date_invoice_range(formatted_date)
+    render json: { "invoice": @invoice, "user": @user, "salary_type": @user.salary_type, "salary_amount": @user.salary, "day_offs_this_month": @user.count_approved_day_offs_by_month(formatted_date) }
```

Comment:
most likely we need to separate this in future. Looks like too many things are returned in one request. 
No need to change anything atm.